src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnector.java

```diff
@@ -79,18 +77,15 @@ public SnowflakeSinkConnector() {
    */
   @Override
   public void start(final Map<String, String> parsedConfig) {
-    // ensure we start counting tasks at 0 for this instance
-    SnowflakeSinkTask.setTotalTaskCreationCount(0);
+    // initialize logging with global instance Id
+    LoggerHandler.setKcGlobalInstanceId(
+        LoggerHandler.getFormattedKcGlobalInstanceId(this.connectorStartTime));
+    LOGGER.info("SnowflakeSinkConnector:starting...");
 
     Utils.checkConnectorVersion();
 
-    LOGGER.info("SnowflakeSinkConnector:start");
     setupComplete = false;
     connectorStartTime = System.currentTimeMillis();
-
-    // initialize logging with global instance Id
-    LoggerHandler.setConnectGlobalInstanceId(this.getKcInstanceId(this.connectorStartTime));
```

Comment:
'SnowflakeSinkConnector:start' log should have the current kc instanceid, so move the instanceid set logic above this log